design-proposals/isolate-emulator-thread-with-smt-fix/isolate-emulator-thread-with-smt-fix.md

```diff
@@ -155,13 +198,20 @@ Will add unit tests to `renderresources_test.go` to check extra housekeeping log
 # Implementation Phases
 ## Phase 1: Align CPU to SMT requirements when feature gate is enabled and annotation is set on the VMI.
 * Virt-config:
-  * Add support for the new feature gate: `FullPCPUsOnly`.
+  * Add support for the new feature gate: `EmulatorThreadEvenParity`.
 * Virt-API:
-  * Add support for the new alpha annotation: `kubevirt.io/CPUManagerPolicyBetaOptions:full-pcpus-only`.
+  * Add support for the new alpha annotation: `kubevirt.io/EmulatorThreadEvenParity:`.
+  * in case annotation is set on kubevirt CR, the feature gate is enabled and the isolateEmulatorThread is enabled:
+    * the VMI mutating-webhook will copy the kubevirt CR annotation to the VMI annotations.
```

Comment:
There should also be a minor change to the VMI creation admitter to also consider the annotation on the Kubevirt CR.

Reply:
true. DONE